pkg/systemvar/var.go

```diff
@@ -0,0 +1,6 @@
+package systemvar
+
+// SystemDefinitonNamespace golbal value for controller and webhook systemlevel namespace
+var (
+	SystemDefinitonNamespace string = "vela-system"
+)
```

Comment:
We already have multiple (too many) directories for this type of file, please find one (under /oam or /utils or controller/common ...) that you like and move this there.